include/llbuild/Basic/CrossPlatformCompatibility.h

```diff
@@ -18,16 +18,47 @@
 #define CrossPlatformCompatibility_h
 
 #ifdef _MSC_VER
-typedef int llbuild_pid_t;
+// Ignore the conflicting min/max defined in windows.h
+#define NOMINMAX
+#include <windows.h>
+typedef HANDLE llbuild_pid_t;
+typedef HANDLE FD;
 #else
 
+#include <unistd.h>
 #if defined(__linux__) || defined(__GNU__)
 #include <termios.h>
 #else
 #include <sys/types.h>
 #endif
 
 typedef pid_t llbuild_pid_t;
+typedef int FD;
+#endif
```

Comment:
Can you please add `// #ifdef _MSC_VER` to the end of this line? I misread this section like 5 times :S